widget_driver/doc/drivers_without_generation.md

```diff
@@ -0,0 +1,60 @@
+# Drivers Without Generation
+
+You do not need to use code generation to use `WidgetDrivers`. You can just as well create all the needed code manually. It is just a bit more manual work for you 😅
+
+It is actually not so much code you have to create for each `driver`. But as you project grows it still adds up.
+
+*So the recommendation would still be to use code generation.*
+
+## What do I need to do
+
+You just need two extra classes!  
+Except for the actual `WidgetDriver` you need to create a `TestDriver` version of your `Driver`. And you need to create a `WidgetDriverProvider` which can create an instance for your `driver` and `testDriver`.
+
+### Creating the TestDriver
+
+The `TestDriver` is used when your widget is created during widgetTests. It cannot have any dependencies to any instance of something. It should just contain hard coded values. *(It can contain static values from some other type.)*
+
+Your `TestDriver` needs to extend the `TestDriver` type and implement your `Driver` type. Here is an example of how to define it:
+
+```dart
+// Imagine you have a driver called `MyCoolWidgetDriver`
+
+class MyCoolWidgetTestDriver extends TestDriver
+    implements MyCoolWidgetDriver {
+  @override
+  String get coolName => 'A cool name';
+
+  @override
+  String get coolImageUrl => CoolImageService.testCoolImageUrl;
+}
+```
+
+Notice in this example that `testCoolImageUrl` is a static property of the `CoolImageService`. Not an instance property.
```

Comment:
Ah, ok. Here you explain it...
Still I would avoid the use of the static in the example code and keep the example code all hardcoded values, to avoid that people just copy+paste the example and don't read the explanation and then use it wrong on their code. This sentence here still could stay and explain to those who read the text, that instead of the hardcoded value in the example code, one could also use a static value from another type.

Reply:
Hmm. Okay. I get it.. people might not read all the docs :D 
I could maybe update the example with a more complex example. Instead of having a url-string, I could show the MyCoolObject-list?

Or should I just skip it?